docs/release_notes/upgrading_to_destinations_v2.md

```diff
@@ -0,0 +1,170 @@
+|-----------------------------------------------	|-----------------	|-------------------------------------------------------------------------------------------------------------	|----------------------	|---------------------	|
+| Successful typing and de-duping ⟶             	| xxx-xxx-xxx     	| {   id: 1,   first_name: “sarah”,   age: 39,   address: {     city: “San Francisco”,     zip: “94131”   } } 	| 2022-01-01 12:00:001 	| 2022-01-01 12:00:00﻿ 	|
+| Failed typing that didn’t break other rows ⟶  	| yyy-yyy-yyy     	| {   id: 2,   first_name: “evan”,   age: “fish”,   address: {     city: “Menlo Park”,     zip: “94002”   } } 	| 2022-01-01 12:00:001 	| 2022-01-01 12:00:00﻿ 	|
+| Not-yet-typed ⟶                               	| zzz-zzz-zzz     	| {   id: 3,   first_name: “edward”,   age: 35,   address: {     city: “Sunnyvale”,     zip: “94003”   } }    	| NULL                 	| 2022-01-01 13:00:00﻿ 	|
+_____________________________
+
+## Deprecating Legacy Normalization
+
+Existing normalization options, both `Raw data (JSON)` and `Normalized tabular data` are now deprecated, and will be unsupported starting Oct 1, 2023.
+
+![Legacy Normalization](./assets/airbyte_legacy_normalization.png)
+
+As a Cloud user, existing syncs will be paused on **Oct 1, 2023**. Follow the upgrade steps below to benefit from Destinations V2.
+
+As an Open Source user, you may choose to upgrade at your convenience. However, destination connector versions prior to Destinations V2 will no longer be supported as of **Oct 1, 2023**.
+
+See [here](to-do-doc) to learn more about Airbyte's breaking change rollout requirements.
+
+### Breakdown of Breaking Changes
+
+Breaking changes will be most notable to users actively syncing data from API sources such as Facebook Marketing, HubSpot, Stripe, Amazon Ads, TikTok Marketing and more. The following table details the delivered data modified by Destinations V2:
+
+| Current Normalization Setting   	 | Source Type                           	| Impacted Data (Breaking Changes)                         	                     |
+|-----------------------------------|---------------------------------------	|--------------------------------------------------------------------------------|
+| Raw JSON                	         | All                                   	| `_airbyte` metadata columns, raw table location                              	 |
+| Normalized tabular data 	         | API Source                            	| Unnested tables, `_airbyte` metadata columns, SCD tables 	                     |
+| Normalized tabular data 	         | Tabular Source (database, file, etc.) 	| `_airbyte` metadata columns, SCD tables                  	                     |
+
+For `destination-bigquery`, we are now loading `JSON` blobs as type `JSON` in BigQuery (introduced last [year](https://cloud.google.com/blog/products/data-analytics/bigquery-now-natively-supports-semi-structured-data)), instead of type `string`.
```

Comment:
```suggestion
Whenever possible, we've taken this opportunity to use the best data type for storing JSON for your querying connivence.  For example, `destination-bigquery` now loads `JSON` blobs as type `JSON` in BigQuery (introduced last [year](https://cloud.google.com/blog/products/data-analytics/bigquery-now-natively-supports-semi-structured-data)), instead of type `string`.
```